packages/compartment-mapper/src/node-modules.js

```diff
@@ -531,33 +558,6 @@ const graphPackages = async (
   return graph;
 };
 
-/**
- * Compute the lexically shortest path from the entry package to each
- * transitive dependency package.
- * The path is a delimited with hashes, so hash is forbidden to dependency
- * names.
- * The empty string is a sentinel for a path that has not been computed.
- *
- * The shortest path serves as a suitable sort key for generating archives that
- * are consistent even when the package layout on disk changes, as the package
- * layout tends to differ between installation with and without devopment-time
- * dependencies.
- *
- * @param {Graph} graph
- * @param {string} location
- * @param {Array<string>} path
- */
-const trace = (graph, location, path) => {
```

Comment:
`path` is now computed in the same pass as graph building. 
Added benefit is: it uses actual package names and not the overrides (from browser field in package.json)